src/FluentValidation/TestHelper/ValidatorTestExtensions.cs

```diff
@@ -70,7 +70,7 @@ public static class ValidationTestExtension {
 			var childValidatorTypes = matchingValidators.OfType<ChildValidatorAdaptor>().Select(x => x.ValidatorType);
 			childValidatorTypes = childValidatorTypes.Concat(matchingValidators.OfType<ChildCollectionValidatorAdaptor>().Select(x => x.ChildValidatorType));
 
-			if (childValidatorTypes.All(x => x != childValidatorType)) {
+			if (childValidatorTypes.All(x => !childValidatorType.GetTypeInfo().IsAssignableFrom(x.GetTypeInfo()))) {

```

Comment:
Use of `GetTypeInfo` is not supported on all of FV's target platforms. Please introduce conditional compilation.

Reply:
I forgot to stage the extra `using` when committing. 
Do you want me to fixup the previous commit or leave as-is?